USAGE.md

```diff
@@ -0,0 +1,307 @@
+
+
+
+# Installation
+
+Write Podfile to your project.
+
+```
+pod 'Userdive-iOS-SDK', '~> 1.0'
+```
+
+Install pods.
+
+```
+pod install
+```
+
+And open workspace.
+
+```
+open YourProject.xcworkspace
+```
+
+
+# Understanding Userdive iOS SDK operation mode
+
+SDK has 2 operation modes below:
+
+* Developer mode
+* Tracking mode
+
+
+## Developer mode
+
+**WARNING: DO NOT USE THIS MODE FOR APPS IN PRODUCTION**
+
+In developer mode, USERDIVE tracks some native views (UIViewController, UIActionSheet, UIAlertView etc.) for heatmap background image.
+Confirmation dialog shows after every transition. Tap `Send` button to upload screen image on USERDIVE.
+
+
+## Tracking mode
+
+USERDIVE tracks user's gesture and transitions in tracking mode.
+Those logs will be uploaded when the app enters to background.
+
+
+# Integrate with your app
+
+## Preparing integration
+
+### Check your Team ID
+
+Check your Team ID at [USERDIVE For Apps website](https://detector.userdive.com/en/apps/).
+
+![team id](http://drive.google.com/uc?export=view&id=0B7UxsiswNc5_cHdTUGpOX0IyU2M)
+
+
+### Check your Bundle ID
+
+Check Bundle ID on Xcode and USERDIVE.
+
+![bundle id settings on USERDIVE](http://drive.google.com/uc?export=view&id=0B7UxsiswNc5_WWVGaFdSMHNFMGs)
+
+![bundle id on Xcode](http://drive.google.com/uc?export=view&id=0B7UxsiswNc5_Tno4VHA4Snh3alE)
+
+## Capturing image
+
+Before using userdive heatmap, capture the images of app view.
+For capturing image, start USERDIVE iOS SDK with developer mode.
+
+
+### Capture images when it detects view transitions
+
+USERDIVE iOS SDK can automatically track user gesture, and view transitions,
+if you use standard iOS UI components (UINavigationController, UITabBarController etc...).
+
+Import USERDIVE iOS SDK header.
+
+```objective-c
+#import "Userdive.h"
+```
+
+Insert `[Userdive startDeveloperMode:<YOUR_TEAM_ID>]` into 
+`AppDeletegate:application:didFinishLaunchingWithOptions:` with your Team ID.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [Userdive setLogEnabled:YES];
+
+  // START DEVELOPER MODE
+  [Userdive startDeveloperMode:<your team id>];
+
+  return YES;
+}
+```
+
+
+### Capture images manually
+
+You need to capture the images of app view manually
+if you use custom view or 3rd party framework like a [Unity](https://unity3d.com/jp).
+
+Import USERDIVE iOS SDK header.
+
+```objective-c
+#import "Userdive.h"
+```
+
+Insert `[Userdive startDeveloperMode:<YOUR_TEAM_ID>]` into 
+`AppDeletegate:application:didFinishLaunchingWithOptions:` with your Team ID.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [Userdive setLogEnabled:YES];
+
+  // START DEVELOPER MODE WITHOUT TRACKERS
+  [Userdive startDeveloperMode:<your team id> trackers:nil];
+
+  return YES;
+}
+```
+
+When you want to capture view is displayed, call the capture method.
```

Comment:
When the screenshot you want to capture is displayed, call the capture method.
ELLIE CHECK